utils/mk-squash-iso.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+help() {
+	cat <<EOF
+mk-squash-iso.sh [options]
+	Turns an sabotage rootfs into an bootable iso.
+	This requires an working isolinux installation on <rootfs>/boot.
```

Comment:
if we copy isolinux.bin by default during extlinux/syslinux installation, it would probably be nice if its going to be copied to boot directly if its found in lib/syslinux/